### DISCUSSION TITLE
All Builds Marked As Queued

### DISCUSSION THREAD

**Discussion by bschrib** [codemagic-ci-cd/codemagic-docs#1588] [Q&A]
Trying to figure out if it's just us or not. I have three apps in my account, and 1 concurrency. This morning any builds that are triggered either manually or by web hook aren't starting but instead being marked as queued. I've checked the builds in each app and there are no currently running builds. We're using m1_mac_mini build agents.

Is anyone else experiencing this issue?

I reached out to Codemagic via support however it indicates they'll be back on Monday.

So far I've tried cancelled all queued builds and started one new build manually.

**Comment by mikhail-tokarev** [comment]
Hi @bschrib and sorry for the delay!

All M1 machines with Xcode 13-14.1 experienced longer queue time due high load yesterday. The issue has been resolved for now. We apologize for the inconvenience caused.
